### PR TITLE
docs: add instruction to replace perseus_integration

### PR DESCRIPTION
**Pull request by Arrekin**
I didn't want to edit your in-depth explanation from the foldable pane, and without that, it would be kind of duplication, so instead, I opted to add a comment to the main.rs example. That way, if someone just copies the code, he'll have information right there, without the need to dig through the page.

I saw other integrations, so I added them, but maybe you don't want them exposed - let me know if you want some tweaking on this.

### CODE REVIEW

File: examples/.base/src/main.rs [hunk 1]
```diff
@@ -3,7 +3,13 @@ mod templates;
 
 use perseus::{Html, PerseusApp};
 
-#[perseus::main(perseus_integration::dflt_server)]
+
+/// Replace `<perseus_integration>` with an integration of your choice.
+/// Examples of supported integrations:
+/// - perseus_warp (use this one if you follow basic tutorials)
```
Comment (arctic-hen7) [inline examples/.base/src/main.rs:9]:
```suggestion
/// - perseus_warp (recommended)
```

**Review by arctic-hen7 — CHANGES_REQUESTED** [review]
Sorry for the late reply, I've been really busy lately! This looks great, just one minor wording thing I'd change, as I've noted.

**Comment by Arrekin** [comment]
Requested change applied.

**Review by arctic-hen7 — APPROVED** [review]
Sorry for the delay, I completely forgot to actually merge this lol!

**Comment by arctic-hen7** [comment]
@Arrekin just so you're aware, I've had to undo your code in fixing #212 by aliasing all instances of `perseus-integration` to `perseus-warp`, which should hopefully render this a non-issue in future!

**Pull request merged**